### PR TITLE
Adding bootstrap instance type for AWS UPI deployments

### PR DESCRIPTION
**Pull request by vavuthu**
Fixes: #5896

Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

File: ocs_ci/framework/conf/default_config.yaml [hunk 1]
```diff
@@ -135,6 +135,8 @@ ENV_DATA:
   base_domain: 'qe.rh-ocs.com'
   master_instance_type: 'm4.xlarge'
   worker_instance_type: 'm5.4xlarge'
+  # vm_type_bootstrap is used in AWS UPI deployments
+  vm_type_bootstrap: 'i3.large'
```
Comment (petr-balogh) [inline ocs_ci/framework/conf/default_config.yaml:139]: Is this some recommended one type for bootstrap? IIRC i3 instances are BM with ssd disks.

Comment (vavuthu) [reply ocs_ci/framework/conf/default_config.yaml:139]: its the default instance type used by OCP QE team

**Comment by vavuthu** [comment]
verified here: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/4259/consoleFull

**Comment by vavuthu** [comment]
```
2022-05-19 12:16:11  06:46:10 - MainThread - ocs_ci.deployment.aws - INFO - Executing UPI install script
2022-05-19 12:19:33  + chmod o+rX -R /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth
2022-05-19 12:57:27  07:27:24 - MainThread - ocs_ci.deployment.aws - INFO - /usr/bin/jq
2022-05-19 12:57:27  /usr/bin/aws
2022-05-19 12:57:27  /home/jenkins/bin/oc
2022-05-19 12:57:27  [INFO] AMI ID: master nodes: ami-01990fc3bdf30bc13
2022-05-19 12:57:27  [INFO] AMI ID: worker nodes: ami-01990fc3bdf30bc13
2022-05-19 12:57:27  Running Command: aws --region us-east-2 cloudformation create-stack --stack-name j-015au3r33-d-vpc   --template-body 'file:///home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/external/openshift-misc-1652942750/functionality-testing/aos-4_11/hosts//upi_on_aws-cloudformation-templates/01_vpc.yaml' --parameters 'file:///home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/external/openshift-misc-1652942750/functionality-testing/aos-4_11/hosts/install-dir/pre_action/aws_vpc_params.json'
2022-05-19 12:57:27  {
2022-05-19 12:57:27      "StackId": "arn:aws:cloudformation:us-east-2:861790564636:stack/j-015au3r33-d-vpc/5f035c50-d73f-11ec-8a7d-0a1cbd081316"
2022-05-19 12:57:27  }
```

**Review by jilju — APPROVED** [review]

**Pull request merged**